src/models/polar/kernels.jl

```diff
@@ -1,11 +1,11 @@
 # Implement kernels for polar formulation
 
 """
-residualFunction
+power_balance
 
 Assembly residual function for N-R power flow
 """
-function residualFunction(V, Ybus, Sbus, pv, pq)
+function power_balance(V, Ybus, Sbus, pv, pq)
```

Comment:
Why did you change this?

Reply:
Good question. Even if this is an external function, this function is being used only in the tests (as it is not compatible with GPU). But, as we renamed `residualFunction_polar` to `residual_polar!`, I thought that it would be more consistent to rename `residualFunction` as well.